src/languages/en.js

```diff
@@ -126,44 +137,71 @@ export default {
     addSecondaryLoginPage: {
         addPhoneNumber: 'Add Phone Number',
         addEmailAddress: 'Add Email Address',
-        enterPreferredPhoneNumberToSendValidationLink: 'Enter your preferred phone number and password to send a validation link.',
-        enterPreferredEmailToSendValidationLink: 'Enter your preferred email address and password to send a validation link.',
+        enterPreferredPhoneNumberToSendValidationLink:
+            'Enter your preferred phone number and password to send a validation link.',
+        enterPreferredEmailToSendValidationLink:
+            'Enter your preferred email address and password to send a validation link.',
         sendValidation: 'Send Validation',
-
     },
     initialSettingsPage: {
         settings: 'Settings',
+        about: 'About',
+        aboutPage: {
+            appDownloadLinks: 'App download links',
+            viewTheCode: 'View the code',
+            viewOpenJobs: 'View open jobs',
+            reportABug: 'Report a bug',
+        },
+        appDownloadLinks: {
+            android: {
+                label: 'Android',
+            },
+            ios: {
+                label: 'iOS',
+            },
+            desktop: {
+                label: 'Desktop',
+            },
+        },
         signOut: 'Sign Out',
         versionLetter: 'v',
         changePassword: 'Change Password',
         readTheTermsAndPrivacyPolicy: {
-            phrase1: 'Read the',
-            phrase2: 'terms of service',
-            phrase3: 'and',
+            phrase1: 'Read the ',
```

Comment:
This makes it inconsistent with the rest of the file, please revert the space changes.

Reply:
Done